virttest/virt_vm.py

```diff
@@ -1216,6 +1216,9 @@ def print_guest_network_info():
                 if serial:
                     break
                 raise
+            except remote.LoginProcessTerminatedError as err:
+                error = err
+                break
             except Exception as err:
                 error = err
```

Comment:
In addition, in order to decrease the times we try to login when an unknown exception is happened, shall we add some sleep here? @xiaodwan

Reply:
I don't think 'sleep' is necessary,  because unknown exception is suppressed by the common Exception clause. If there are other scenarios are not considered,  a sleep will not help too much, and it may hidden some real script bugs.

Reply:
I do not agree this will hid some script bugs. This suggestion is for your statement below:
```
 But for some unknow reason, the ssh command timeout, and handle_prompts
raise a timeout exception, and in the while loop of wait_for_login, the exception errors was only
saved silently and then try self.login again.
```
In your test log,  there are many times to log in 'Attempting to log into'  at 2020-02-23 23:30:43 , so many logs are printed. I think we can add sleep to avoid of printing so many logs in case there is no LoginProcessTerminatedError happened, ssh timeout exception will always happen, then the  'Attempting to log into'  log will print too many.

Reply:
The remote_login has already had a timeout function by default, this can protect the script will not print too much 'Attempting to log into' and it is how it works before. I don't see any 'Attempting to log into' logs in my experience. If add another sleep function, it will delay the script running normally. 
And for other uncaught exceptions, I can't think of such an exception that will block login operation.

Reply:
If that is the case, I can agree so far.